server/public/javascript/sanitize.js

```diff
@@ -0,0 +1,15 @@
+const sanitizeEmails = (customers) => {
+
+  for (const customer of customers) {
```

Comment:
Hmmm customers is an array right? Try to use the new ES6 functions like[ **map**.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)